app/src/main/java/com/rage/clamber/Activities/WallActivity.java

```diff
@@ -1,27 +1,94 @@
 package com.rage.clamber.Activities;
 
+import android.os.Bundle;
 import android.support.v4.app.FragmentTransaction;
 import android.support.v7.app.AppCompatActivity;
-import android.os.Bundle;
+import android.widget.ImageButton;
 
 import com.rage.clamber.Fragments.ActionBarFragment;
+import com.rage.clamber.Fragments.WallSectionFragment;
 import com.rage.clamber.R;
+import com.squareup.picasso.Picasso;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
 
 /**
  * The WallActivity contains data about each wall in the climbing gym.
  */
 public class WallActivity extends AppCompatActivity {
 
+    @Bind(R.id.wall_page_wall_1_button)
+    ImageButton wall1Button;
+    @Bind(R.id.wall_page_wall_2_button)
+    ImageButton wall2Button;
+    @Bind(R.id.wall_page_wall_3_button)
+    ImageButton wall3Button;
+    @Bind(R.id.wall_page_wall_4_button)
+    ImageButton wall4Button;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_wall);
+        ButterKnife.bind(this);
 
         //Sets ActionBar to top of activity screen.
         FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
         transaction.replace(R.id.wall_page_frame_layout, ActionBarFragment.newInstance());
         transaction.commit();
+
+        //Sets the images for the image buttons.
+
```

Comment:
For this Walls Page, I don't think that I will need to change this. It will just be these 4 images. The Walls Section page with 10+ images will not follow this same pattern - those images will be created in the recyclerview adapter.